core/lib/presentation/views/text/rich_text_builder.dart

```diff
@@ -80,7 +83,13 @@ class _RichTextBuilderState extends State<RichTextBuilder> with AutomaticKeepAli
     return SingleChildScrollView(
       physics: const NeverScrollableScrollPhysics(),
       scrollDirection: Axis.horizontal,
-      child: text,
+      child: VisibilityDetector(
+        key: ValueKey(hashCode),
```

Comment:
Is it dangerous to use `hashcode` like that? What if 2 different objects have the same hashcode? If the `RichTextBuilder` object changes frequently it can lead to unnecessary widget re-rendering.

Reply:
Removed